sklearn/notebooks/ml_engine_training.ipynb

```diff
@@ -0,0 +1,501 @@
+    "* `runtime-version` - The version of Cloud ML Engine to use for the job. If you don't specify a runtime version, the training service uses the default Cloud ML Engine runtime version 1.0. See the list of runtime versions for more information.\n",
+    "* `python-version` - The Python version to use for the job. Python 3.5 is available with runtime version 1.4 or greater. If you don't specify a Python version, the training service uses Python 2.7.\n",
+    "* `scale-tier` - A scale tier specifying the type of processing cluster to run your job on. This can be the CUSTOM scale tier, in which case you also explicitly specify the number and type of machines to use.\n",
+    "\n",
+    "Note: Check to make sure gcloud is set to the current PROJECT_ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Updated property [core/project].\n",
+      "\n",
+      "\n",
+      "Updates are available for some Cloud SDK components.  To install them,\n",
+      "please run:\n",
+      "  $ gcloud components update\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud config set project $PROJECT_ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Submit the training job."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job [census_training_20180605_150110] submitted successfully.\r\n",
+      "Your job is still active. You may view the status of your job with the command\r\n",
+      "\r\n",
+      "  $ gcloud ml-engine jobs describe census_training_20180605_150110\r\n",
+      "\r\n",
+      "or continue streaming the logs with the command\r\n",
+      "\r\n",
+      "  $ gcloud ml-engine jobs stream-logs census_training_20180605_150110\r\n",
+      "jobId: census_training_20180605_150110\r\n",
+      "state: QUEUED\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! gcloud ml-engine jobs submit training census_training_$(date +\"%Y%m%d_%H%M%S\") \\\n",
+    "  --job-dir $JOB_DIR \\\n",
+    "  --package-path $TRAINER_PACKAGE_PATH \\\n",
+    "  --module-name $MAIN_TRAINER_MODULE \\\n",
+    "  --region $REGION \\\n",
+    "  --runtime-version=$RUNTIME_VERSION \\\n",
+    "  --python-version=$PYTHON_VERSION \\\n",
+    "  --scale-tier BASIC"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# [Optional] StackDriver Logging\n",
+    "You can view the logs for your training job:\n",
+    "1. Go to https://pantheon.corp.google.com \n",
```

Comment:
change to cloud console url before merge

Reply:
Right!